src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc") format("ttc"),
+    url("fonts/Avenir-Book.ttf") format("truetype"),
+    url("fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+body {
+  font-weight: 300;
+}
+
+.bar {
+  display: block;
+}
+
+.first-bar_form {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.first-bar {
+  width: 100%;
+  background: url(images/Search.svg) no-repeat 24px 24px #fff;
+  background-size: 19px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  padding: 23px;
+  padding-left: 62px;
+
+  font-family: "Avenir", Helvetica, sans-serif;
+  line-height: 22px;
+  font-size: 16px;
+}
+
+.first-bar::placeholder,
+.second-bar::placeholder {
+  color: #3d4e61;
+}
+
+.first-bar:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.first-bar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.second-bar {
+  width: 100%;
```

Comment:
it is bad naming for classes
avoid using first, second, numbers
use common classes for first bar and with `--small` modifier for second one